v63003/instream.csh

```diff
@@ -20,14 +20,16 @@
 # gtm8804	    [vinay] Test zshow "t" produces only a summary of zshow "g" and "l"
 # gtm8832	    [vinay] Test string literal evaluating to >=1E47 produces a NUMOFLOW error
 # gtm8617	    [vinay] Tests MUPIP SET command of STDNULLCOLL and NULL_SUBSCRIPTS
+# gtm8732nr	    [vinay] Tests the valid inputs of the flag DEFER_TIME for the MUPIP SET command
```

Comment:
Please add a line like this (that describes what the test is trying to accomplish) in the first line of the reference file for both subtests.